docker-compose.yaml

```diff
@@ -266,32 +267,39 @@ services:
 #      timeout: 20s
 #      retries: 3
 
-#  milvus:
-#    # Turn on to leverage the `vdb_upload` task
-#    restart: always
-#    container_name: milvus-standalone
-#    image: milvusdb/milvus:v2.3.5
-#    command: ["milvus", "run", "standalone"]
-#    hostname: milvus
-#    security_opt:
-#      - seccomp:unconfined
-#    environment:
-#      ETCD_ENDPOINTS: etcd:2379
-#      MINIO_ADDRESS: minio:9000
-#    volumes:
-#      - ./.volumes/milvus:/var/lib/milvus
-#    healthcheck:
-#      test: ["CMD", "curl", "-f", "http://localhost:9091/healthz"]
-#      interval: 30s
-#      start_period: 90s
-#      timeout: 20s
-#      retries: 3
-#    ports:
-#      - "19530:19530"
-#      - "9091:9091"
-#    depends_on:
-#      - "etcd"
-#      - "minio"
+  # milvus:
+  #   # Turn on to leverage the `vdb_upload` task
+  #   restart: always
+  #   container_name: milvus-standalone
+  #   image: milvusdb/milvus:v2.4.9-gpu
```

Comment:
This is the new version of milvus that we would need to use for GPU indexing to work.